### PR TITLE
#699 Panel pack. Include PQE for leadership exercises

### PR DESCRIPTION
**Pull request by warrensearle**
Currently PQE is only included in the exported Application Data google documents for exercises which have type `legal`.

Here we are including PQE for exercises of type `leadership`.

Also provides package scripts for deploying individual functions, as follows:
`npm run staging:deploy:function #function-name#`

### CODE REVIEW

File: nodeScripts/initPanelExport.js [hunk 1]
```diff
@@ -9,11 +9,12 @@
...
 
-const { app } = require('./shared/firebase.js');
-const { initialisePanelExport } = require('../functions/actions/panels/initialisePanelExport');
+const config = require('./shared/config');
+const { firebase, app, db } = require('./shared/admin.js');
+const { initialisePanelExport } = require('../functions/actions/panels/initialisePanelExport')(config, firebase, db);
 
 const main = async () => {
-  await initialisePanelExport('25GTawkJ0ic1XLYKG29B');
+  await initialisePanelExport('ozUWQ7ewwnA0RZOZ8nIP');
```
Comment (tomlovesgithub) [inline nodeScripts/initPanelExport.js:17]: is it an issue that this has changed?

File: nodeScripts/processPanelExport.js [hunk 2]
```diff
@@ -9,11 +9,12 @@
...
 
-const { app } = require('./shared/firebase.js');
-const { processPanelExport } = require('../functions/actions/panels/processPanelExport');
+const config = require('./shared/config');
+const { firebase, app, db } = require('./shared/admin.js');
+const { processPanelExport } = require('../functions/actions/panels/processPanelExport')(config, firebase, db);
 
 const main = async () => {
-  await processPanelExport('25GTawkJ0ic1XLYKG29B');
+  await processPanelExport('ozUWQ7ewwnA0RZOZ8nIP');
```
Comment (tomlovesgithub) [inline nodeScripts/processPanelExport.js:17]: same here

File: nodeScripts/initPanelExport.js (hunk 1 above)
Comment (warrensearle) [reply nodeScripts/initPanelExport.js:17]: No, that's fine @tomlovesgithub the node script is just for testing out the functionality from local CLI

File: nodeScripts/processPanelExport.js (hunk 2 above)
Comment (warrensearle) [reply nodeScripts/processPanelExport.js:17]: No, that's fine @tomlovesgithub the node script is just for testing out the functionality from local CLI

**Review by tomlovesgithub — APPROVED** [review]

**Review by HalcyonJAC — APPROVED** [review]
Code looks great!

**Pull request merged**